ruby-magic-link-example/views/check_email.erb

```diff
@@ -1,13 +1,39 @@
-  <div class="container_login">
-    <div class='flex_column'>
-    <div class="flex heading_div">
-        <img src="<%= url("/workos_logo_new.png")%>" alt="workos logo">
-        <div class="heading_text_div">
-            <h1>WorkOS</h1>
+<div class="logged_in_nav">
+        <div class="flex">
+            <div>
+                <img src="<%= url("images/workos-logo-with-text.png")%>" alt="workos logo">
```

Comment:
Could you remove the border from the logo on the upper left? 
![CleanShot 2022-08-12 at 12 38 26](https://user-images.githubusercontent.com/47123183/184422846-a53f1ad1-e2ca-4afd-b66d-1fa892c3ffee.png)

Reply:
Sinatra CSS browser caching related.  Should be all good
<img width="1651" alt="CleanShot 2022-08-15 at 08 55 15@2x" src="https://user-images.githubusercontent.com/96083517/184670114-b7bea995-eaec-4d83-98ce-5e89c75051a0.png">
!